osf_pigeon/app.py

```diff
@@ -0,0 +1,73 @@
+import logging
+import requests
+from osf_pigeon import pigeon
+from concurrent.futures import ThreadPoolExecutor
+from osf_pigeon import settings
+from aiohttp import web
+
+import sentry_sdk
+from sentry_sdk.integrations.aiohttp import AioHttpIntegration
+
+sentry_sdk.init(
+    dsn=settings.SENTRY_DSN,
+    release="0.0.10",
+    integrations=[AioHttpIntegration()],
+)
+
+pigeon_jobs = ThreadPoolExecutor(max_workers=1, thread_name_prefix="pigeon_jobs")
+app = web.Application()
+routes = web.RouteTableDef()
+logging.basicConfig(filename="pigeon.log", level=logging.DEBUG)
+
+
+def handle_exception(future):
```

Comment:
maybe some needless DRY happening by having a separate callback instead of doing exception handling in the main callback for each function.